packages/abi-gen-templates/Python/contract.handlebars

```diff
@@ -11,13 +20,55 @@ from zero_ex.contract_wrappers._base_contract_wrapper import BaseContractWrapper
 from zero_ex.contract_wrappers.tx_params import TxParams
 
 
+class {{contractName}}ValidatorBase:
```

Comment:
Well, I _could_ do that, but...
-   that would mean a not insignificant change to `abi-gen`, to start outputting more than one file, and I'm feeling like I'd rather see this PR merge and follow up later as necessary; but also
-   it's not uncommon in Python land to have multiple classes in a file, _especially_ in an `__init__.py` file, which represents the "root" of the Python module named after its parent folder.  we're now writing the wrapper to `exchange/__init__.py`, not `exchange.py`, and people generally know that if you're looking for the definition of a class, and you don't see a file for it, it must be in `__init__.py`.

Maybe @PirosB3 can weigh in on my assessment of the Pythonic style here...

Reply:
Oh you mean it's own _handlebars_ file, I mis-read, sorry.  I think my logic still applies though, in that I think it makes sense to structure the template files analogous to the generated files.  In TypeScript there's a conventional 1:1 mapping between files and classes, but it may not be that way in every target language environment, and it's not in Python.  I think it makes sense to structure the templates analogous to the files they will produce.  (I don't feel _that_ strongly about it, but that's my logic.)